consensus/module.go

```diff
@@ -194,13 +193,24 @@ func (m *consensusModule) Start() error {
 		return err
 	}
 
-	go m.metadataSyncLoop()
-	go m.blockApplicationLoop()
+	m.wg.Add(1)
+	go func() {
+		defer m.wg.Done()
+		m.blockApplicationLoop()
+	}()
 
 	return nil
 }
 
 func (m *consensusModule) Stop() error {
+	close(m.blocksResponsesReceived)
```

Comment:
1. Is this the only channel we need to close?
2. Is there a way to log if there are still other messages inside it when we close it?